Modules/HTTP/Tests/HTTPTests/Message/RequestTests.swift

```diff
@@ -10,16 +10,16 @@ public class RequestTests : XCTestCase {
         XCTAssertEqual(request.headers, ["Content-Length": "0"])
         XCTAssertEqual(request.body, .buffer(Buffer()))
 
-        request = Request(body: Drain(buffer: "foo") as Core.InputStream)
+        request = Request(body: BufferStream(buffer: "foo"))
         XCTAssertEqual(request.method, .get)
         XCTAssertEqual(request.url, URL(string: "/"))
         XCTAssertEqual(request.version, Version(major: 1, minor: 1))
         XCTAssertEqual(request.headers, ["Transfer-Encoding": "chunked"])
         XCTAssertTrue(request.body.isReader)
 
         request = Request { stream in
-            try stream.write("foo")
-            try stream.flush()
+            try stream.write("foo", deadline: 1.second.fromNow())
```

Comment:
Of course we will add doc comments to all APIs explaining that deadlines are absolute, and because of that they need the `fromNow()`. The reason they're absolute is (quoting libmill):

> One thing to note is that libmill uses deadlines rather than more conventional timeouts. In other words, you specify the time instant when you want the operation to finish rather than maximum time it should take to run.

> The advantage of deadlines over timeouts is that a single deadline can be reused in many subsequent function calls without having to worry about readjusting its value to account for time already elapsed since the deadline was created.

Another important thing to notice is that deadlines are most of the times timeout parameters in which you create a deadline and then reuse that deadline across all the sub-operations. So the call site most of the times doesn't even use `1.second.fromNow()` that's mostly used in tests. Actually the tests are being stupid not to reuse the same deadline. I'll fix that.